docs/user-guide/how_to.ipynb

```diff
@@ -59,23 +59,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sc.Variable(value=[1,2,3])"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Initialized with default values"
```

Comment:
Why removed? Shouldn't this mention `zeros`?

Reply:
Didn't make sense given that `zeros` was restricted to the actual number 0. Adding it back in now.